### ISSUE TITLE
Cobbleswarm Action Issue

### ISSUE THREAD

**Issue by computerpro82** [foundryvtt/pf2e#11576]
The Cobbleswarm as found in PF2 Scenario 3-16 Escape from Oppara has an issue. When I try to use their "Grasping Bites" action, I just get multiple repetitions of my username in the Chat window. I am currently running on V11 patch 13 build 325. This happens on two different tables, and I replaced the actors from the bestiary browser without success.

**Comment by simonward** [comment]
I'm not able to reproduce. Do you have steps? Can you reach out on Discord?

**Comment by computerpro82** [comment]
I am assuming you mean the Foundry VTT Discord server and the PF2e
channel under Game Systems?  Yes, I have Discord access.

All I did yesterday was to import the Cobbleswarm from the Bestiary
browser. AS soon as I tried to use either of the abilities, my userid is
spammed into the chat and I never see the action effect. This happens
whether I invoke the activity from the HUD or the character sheet.

I have two tables on The Forge that are having this problem. I can provide
the link and once someone has access to the table, I can give ownership of
the Cobbleswarm and we can see if the issue replicates on the table.
--
Eric S. Parker
719-481-0786(h)
719-339-6509(c)
Quote from Babylon 5, US Television show, originally aired 1993-1998"Mr.
Garibaldi, there're days I'm very glad I don't have to think the way you
do."
*Ivanova*, *And the Sky Full of Stars*

“There is a cult of ignorance in the United States, and there has always
been. The strain of anti-intellectualism has been a constant thread winding
its way through our political and cultural life, nurtured by the false
notion that democracy means that 'my ignorance is just as good as your
knowledge.”
― Isaac Asimov

**Comment by simonward** [comment]
Did you try disabling modules?

**Comment by computerpro82** [comment]
I just did. I managed to narrow the issue to the PF2E Workbench module. I
had version 5.32.4 installed. I just upgraded to version 5.34.0 and the
issue is still occurring.
--
Eric S. Parker
719-481-0786(h)
719-339-6509(c)
Quote from Babylon 5, US Television show, originally aired 1993-1998"Mr.
Garibaldi, there're days I'm very glad I don't have to think the way you
do."
*Ivanova*, *And the Sky Full of Stars*

“There is a cult of ignorance in the United States, and there has always
been. The strain of anti-intellectualism has been a constant thread winding
its way through our political and cultural life, nurtured by the false
notion that democracy means that 'my ignorance is just as good as your
knowledge.”
― Isaac Asimov
